src/fontra/views/editor/panel-reference-font.js

```diff
@@ -59,7 +61,44 @@ function cleanFontItems(fontItems) {
     return {
       uplodadedFileName: fontItem.uplodadedFileName,
       fontIdentifier: fontItem.fontIdentifier,
+      supportedLanguages: fontItem.supportedLanguages,
+    };
+  });
+}
+
+async function readSupportedLanguages(fontItem, languageMapping) {
```

Comment:
```suggestion
function readSupportedLanguages(fontItem, languageMapping) {
```
You are returning a promise, so it is async in that way, and there is no need to declare this func as async.

Reply:
Thanks. Previously I was using await inside that function. Looks like I forgot to delete async keyword when it evolved to the current version.